rpcs3/Emu/Io/LogitechG27.h

```diff
@@ -112,9 +122,17 @@ class usb_device_logitech_g27 : public usb_device_emulated
 
 private:
 	void sdl_refresh();
+	void set_personality(logitech_personality, bool reconnect = false);
+	void transfer_df(u32 buf_size, u8* buf, UsbTransfer* transfer);
+	void transfer_dfp(u32 buf_size, u8* buf, UsbTransfer* transfer);
+	void transfer_dfgt(u32 buf_size, u8* buf, UsbTransfer* transfer);
+	void transfer_g25(u32 buf_size, u8* buf, UsbTransfer* transfer);
+	void transfer_g27(u32 buf_size, u8* buf, UsbTransfer* transfer);
 
 	u32 m_controller_index = 0;
 
+	logitech_personality m_personality;
+	u8 m_next_id = 0xff;
```

Comment:
Why not just use the enum you added and add something like invalid instead of using this confusing number casting?